woocommerce/woocommerce-ios
# Aztec editor for Product description: part 2.2 - format bar actions - ordered/unordered list, heading

Part 2.2 for #1421 

## Changes
- Added 2 format bar commands that present a list of options:
  - `AztecHeaderFormatBarCommand` with basic tests
  - `AztecUnorderedListFormatBarCommand` with basic tests (no public access to the options)
- Added `OptionsTableViewController+Styles` to style the options table view
- DI'ed `OptionsTablePresenter` to `AztecFormatBarCommandCoordinator`
- Allowed testing Aztec editor when tapping Edit on Product Details screen

## Testing
- Go to Products tab
- Tap a Product
- Tap "Edit" in the navigation bar
- Play with "H" header and unordered list format bar actions --> they should format the selected text / active text as in WPiOS

## Example screenshots

@kyleaparker we can do a design review after the major Aztec steps are done

Heading options | list options
-- | --
![Simulator Screen Shot - iPhone SE - 2019-11-12 at 12 06 00](https://user-images.githubusercontent.com/1945542/68641749-bb831100-0546-11ea-8014-a53da69f41eb.png) | ![Simulator Screen Shot - iPhone SE - 2019-11-12 at 12 06 39](https://user-images.githubusercontent.com/1945542/68641750-bb831100-0546-11ea-9d83-27a3ae51e37b.png)


Update release notes:
